diff -urN node-v0.5.1.orig/tools/wafadmin/Tools/node_addon.py node-v0.5.1/tools/wafadmin/Tools/node_addon.py
--- node-v0.5.1.orig/tools/wafadmin/Tools/node_addon.py	2011-07-14 23:10:36.000000000 +0000
+++ node-v0.5.1/tools/wafadmin/Tools/node_addon.py	2011-07-16 18:43:00.000000000 +0000
@@ -28,7 +28,7 @@
   lib = join(prefix, 'lib')
 
   conf.env['LIBPATH_NODE'] = lib
-  conf.env['CPPPATH_NODE'] = join(prefix, 'include', 'node')
+  conf.env['CPPPATH_NODE'] = join(prefix, 'include', 'nodejs')
 
   conf.env.append_value('CPPFLAGS_NODE', '-D_GNU_SOURCE')
 
@@ -49,7 +49,7 @@
   found = os.path.exists(conf.env['NODE_PATH'])
   conf.check_message('node path', '', found, conf.env['NODE_PATH'])
 
-  found = os.path.exists(join(prefix, 'bin', 'node'))
+  found = os.path.exists(join(prefix, 'bin', 'nodejs'))
   conf.check_message('node prefix', '', found, prefix)
 
   ## On Cygwin we need to link to the generated symbol definitions
@@ -63,7 +63,7 @@
     nodePath = None
     if not os.environ.has_key('NODE_PATH'):
         if not os.environ.has_key('HOME'):
-            nodePath = join(get_prefix(), 'lib', 'node')
+            nodePath = join(get_prefix(), 'lib', 'nodejs')
         else:
             nodePath = join(os.environ['HOME'], '.node_libraries')
     else:
@@ -73,7 +73,7 @@
 def get_prefix():
     prefix = None
     if not os.environ.has_key('PREFIX_NODE'):
-        prefix = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', '..'))
+        prefix = '/usr'
     else:
         prefix = os.environ['PREFIX_NODE']
     return prefix
